### PR TITLE
Add build tests for different ROS versions

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Problem Description**
This commit adds a simple build test for the following ROS versions
- ROS Melodic
- ROS Noetic

**Additional Context**
- The package doesn't seem to work with ROS Noetic yet from https://github.com/ethz-asl/versavis/issues/21, but the catkin package still builds.

### CODE REVIEW

**Comment by Jaeyoung-Lim** [comment]
The CI runs on my fork, but not here: https://github.com/Jaeyoung-Lim/versavis/pull/1 @floriantschopp Is the repo somehow configured not to run github actions?

File: .github/workflows/build_test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {rosdistro: 'melodic', container: 'px4io/px4-dev-ros-melodic:2020-08-14'}
+          - {rosdistro: 'noetic', container: 'px4io/px4-dev-ros-noetic:2020-08-20'}
```
Comment (floriantschopp) [inline .github/workflows/build_test.yml]: could we also use a standard repo here instead px4io?

File: .github/workflows/build_test.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: Build Test
+on:
+  push:
+    branches:
+    - 'master'
+  pull_request:
+    branches:
+    - '*'
```
Comment (floriantschopp) [inline .github/workflows/build_test.yml:10]: Are there also nightly tests?

**Review by floriantschopp — CHANGES_REQUESTED** [review]
@Jaeyoung-Lim Thanks for adding this. I am new to GitHub releases, so not much experience there. Since VersaVIS depends on a lot more than just a successful build of the ROS packages, adding such tests was quite low priority. But it also can't hurt :-)

File: .github/workflows/build_test.yml (hunk 2 above)
Comment (Jaeyoung-Lim) [reply .github/workflows/build_test.yml:10]: Yes!

Comment (floriantschopp) [reply .github/workflows/build_test.yml:10]: I think we should add this, then I am fine for trying that change...

Comment (Jaeyoung-Lim) [reply .github/workflows/build_test.yml:10]: Done! I have configured it to run 18:00 UTC everyday

**Comment by Jaeyoung-Lim** [comment]
@floriantschopp Can we enable github actions for this repo? Then I can also add build status badges to the readme in this PR

File: .github/workflows/build_test.yml (hunk 1 above)
Comment (Jaeyoung-Lim) [reply .github/workflows/build_test.yml]: Yeap, I think that makes more sense

Comment (Jaeyoung-Lim) [reply .github/workflows/build_test.yml]: Done!

**Review by floriantschopp — APPROVED** [review]
lgtm, thanks!!

**Pull request merged**